notebooks/tutorial/02_performance.ipynb

```diff
@@ -0,0 +1,478 @@
+{
+ "cells": [
```

Comment:
Spacy batch mode should be added to provide "best case" comparison

Reply:
Agreed. I'm interested in seeing the performance if this as well.